vagrant/centos7example/README.md

```diff
@@ -0,0 +1,20 @@
+# vagrant
+
+We are currently using vagrant's private networking option. This means we can assign the local IP address and access from Chrome/DevTools.
+It also means we don't have to specifically forward every port that we may want at a later date for each internal Docker container.
+https://www.vagrantup.com/docs/networking/private_network.html
+
+The vagrantfile included here is a basic Centos7 box setup.
+The shell instructions which will be included, will automatically provision the Docker and CKAN environment. 
+
+## Note:
+````
+It might be wise to leave the following command out of the shell script due to how long it takes to run:
+sudo docker-compose up -d --build
```

Comment:
If we have pre-built images of CKAN that we can download over a fast network, then even on test VMs we can integrate dockerhub etc. into shell scripts or ansible provisioning etc.